### PR TITLE
Issue #336: added retransmission of repair messages

### PR DESCRIPTION
**Pull request by carllin**
https://github.com/solana-labs/solana/issues/336

### CODE REVIEW

File: src/crdt.rs [hunk 1]
```diff
@@ -590,33 +590,45 @@ impl Crdt {
...
+            let blob_ix = wblob.get_index().expect("run_window_request get_index");
             if blob_ix == ix {
+                let num_retransmits = wblob.meta.num_retransmits;
+                wblob.meta.num_retransmits += 1;
+                
+                if me.current_leader_id == me.id &&
+                   num_retransmits != 0 && 
+                   !num_retransmits.is_power_of_two()
+                {
+                    return None;
```
Comment (pgarg66) [inline src/crdt.rs]: If it's not power of two, shouldn't we transmit with id of receiver (the original logic)?

Comment (carllin) [reply src/crdt.rs]: good catch, will update soon

Comment (pgarg66) [reply src/crdt.rs]: Cool, while at it, could you fix the code formatting error (causing CI failure)

Comment (carllin) [reply src/crdt.rs]: Hm, I misunderstood this part of the original issue. Even if the goal is to reduce network congestion among the validators, it still does make more sense to just respond with a response, but not rebroadcast, on non-power of two requests. Good point!

Comment (carllin) [reply src/crdt.rs]: Sure, done and done :)

**Comment by garious** [comment]
See #382

**Pull request closed without merging**